sdk/samples/iot/aziot_realtek_amebaD/create_trusted_cert_header.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
```

Comment:
```bash
set -x # Set trace on
set -o errexit # Exit if command failed
set -o nounset # Exit if variable not set
set -o pipefail # Exit if pipe failed
```

Reply:
Ewerton, I believe you are using a copied version of the scripts found in aziot_esp8266, no?

What if this was a softlink to a standard script, so that any changes (like Dane's suggestion), only has to be made in one location.  All sample directories needing to use the scripts keep a softlink.

Reply:
Done.